platform-stack/docker-compose.yml

```diff
@@ -56,6 +56,30 @@ services:
     networks:
       - internal
 
+  deployment:
+    image: miragon/miranum-deployment-service:latest
+    container_name: miranum-deployment
+    ports:
+      - "8092:8080"
+    profiles:
+      - deployment
```

Comment:
Why did you use an profile for that? Shouldn't the deployment-service always be part of our stack?

Reply:
Good point. I will remove the profile. Thanks for the feedback